bioptim/interfaces/acados_interface.py

```diff
@@ -257,7 +258,8 @@ def __update_solver(self):
 
         if self.y_ref_end:
             self.ocp_solver.cost_set(self.acados_ocp.dims.N, "yref", np.concatenate([data for data in self.y_ref_end]))
-            self.ocp_solver.cost_set(self.acados_ocp.dims.N, "W", self.W_e)
+            # check following line
+            # self.ocp_solver.cost_set(self.acados_ocp.dims.N, "W", self.W_e)
```

Comment:
Why did you comment this instead of removing?

Reply:
idem